projects/roade/roade.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Roade
+description: Check out how we helped Roade in building an application that helps intracity commuting for people in the best and efficient way. Get the best React Native developers to build your product!
+meta:
+- react native solution provider
+- react native services
+- node.js solutions provider
+- node.js services
+keywords:
+- roade
+- react native development company
+- react native development company in san francisco
+- react native development companies in bangalore
+- react native dev shop in USA
+- node.js developement company
+- node.js development company in san francisco
+- node.js development companies in bangalore
+- node.js dev shop in USA
+link: /roade
+---
+
+In 2018, Mystro Founder Herb approached Codebrahma for a new product roadmap. They wanted to build an application that helps intracity commuting for people in the best and efficient way.
+
+We build the complete stack which includes backend server, front end apps for both iOS and Android.
+
+![Roade Screenshot](./images/Screenshot_20190715-122615.png)
+
+![Roade Search Screenshot](./images/Screenshot_20190715-122659.png)
+
+## Challenge
+
+The major challenge in implementing the idea like Bestride is the different types of data format which each of the services like Uber, Lyft and other ride-hailing apps return in their APIs. The app has to handle all the types of data structures without compromising the performance of the app. In addition to that, the app should stand out from the giant competitors already in the market which is achieved without any doubts. 
+
+## Solution
+Deciding on what computation needs to be on the server and what needs to be done on the client app is very critical for this project. The team has done a tremendous job in architecting the app so that there is a minimal change required to do whenever any of the API providers change their data structure.
+
+![Roade Search Screenshot](./images/Screenshot_20190715-122940.png)
+
+![Roade Uber Ride Screenshot](./images/Screenshot_20190715-122728.png)
+
+The mobile application is optimized to the level where even on continuous use of services like geolocation and a significant count of API requests made parallel at any time, the app seems to be responsive, robust and reliable.
+
+![Roade Ride Details Screenshot](./images/Screenshot_20190715-122915.png)
+
+## Tech Stack: 
+ * Node Js for server
+ * React Native for mobile app (Android and iOS)
```

Comment:
Can you replace the text with the below.
```
In 2018, Mystro Founder Herb approached Codebrahma for a new product roadmap. They wanted to build an application that helps intracity commuting for people in the best and efficient way.

We built the complete stack which includes back end(written in Node.js) and front end(written in React Native) for both iOS and Android.

Challenges:
The major challenge in implementing the idea like Bestride is, the data provided by the APIs of services like Uber, Lyft and other ride-hailing apps. Each service provides the ride data in different format. The app has to handle all the types of data structures without compromising the performance. In addition to that, the app should stand out from the giant competitors already in the market. 

Taking a careful decision on what computation needs to be on the server and what needs to be done on the client app is very critical for this project. The team has done a tremendous job in architecting the app so that there is a minimal change required to do whenever any of the API providers change their data structure.

The mobile application is optimised to the level where even on continuous use of services like geolocation and a significant count of API requests made parallel at any time, the app seems to be responsive, robust and reliable.

Tech stack
* Node Js for server
* React Native for mobile app (Android and iOS)


```